components/Tabs/examples/supplemental/TabBarIconExampleSupplemental.h

```diff
@@ -29,11 +29,11 @@
 
 @interface TabBarIconExample : UIViewController <MDCTabBarDelegate>
 
-@property MDCTabBar *_Nullable tabBar;
-@property MDCRaisedButton *_Nullable alignmentButton;
-@property MDCAppBar *_Nullable appBar;
-@property UIScrollView *_Nullable scrollView;
-@property UIView *_Nullable starPage;
+@property(nonatomic) MDCTabBar *_Nullable tabBar;
```

Comment:
super-nit: If we are touching these lines, can we move _Nullable into the @property annotation?
@property(nonatomic, nullable) MDCTabBar *tabBar;